### ISSUE TITLE
Downloading likes

### ISSUE THREAD

**Issue by mbarry84** [johanneszab/TumblThree#74]
I am wanting to download my likes as well as other blogs. Do I need to just follow the add new website crawler directions to add my likes or is it better to try and implement it in the existing program.

**Comment by johanneszab** [comment]
Do it how you like to do it.

My guess it's probably better to have a separated task for that since you cannot use the old api for that since it has no way to authenticate yourself to the network.

You might want to take a look at the parse branch since I've used the webbrowser control (internet explorer) for authentication there. I don't know if thats a good way at all. If you come up with something better, go for it.

We might have to add a new _kind_ column in the datagrid (blog manager on the left) for different sites anyways so that you can sort for _tumblr_, _twitter_ etc. 

Basically feel free to do anything ..

**Comment by johanneszab** [comment]
I've implemented a quick hack. It downloads and seems to work, but it's probably a good idea to test things a bit more before releasing a new version.

For downloading liked photos and videos (more isn't implemented yet), you have to do some steps:
1. Go to Settings, click the _Authenticate_ button. Logon to tumblr using an account. TumblThree will use the Internet Explorer cookies for authentication.
2. Add the blog url including the _liked/by_ string in the url (e.g. https://www.tumblr.com/liked/by/wallpapferfx/).
